src/ExpressionEvaluator/CSharp/Test/ResultProvider/AccessibilityTests.cs

```diff
@@ -88,20 +88,19 @@ class C
             var bytesB = compilationB.EmitToArray();
             var assemblyA = ReflectionUtilities.Load(bytesA);
             var assemblyB = ReflectionUtilities.Load(bytesB);
-            DkmClrValue valueWithNonPublicHidden;
+            DkmClrValue value;
 
             using (ReflectionUtilities.LoadAssemblies(assemblyA, assemblyB))
             {
                 var runtime = new DkmClrRuntimeInstance(new[] { assemblyB });
                 var type = assemblyB.GetType("C", throwOnError: true);
-                valueWithNonPublicHidden = CreateDkmClrValue(
+                value = CreateDkmClrValue(
                     Activator.CreateInstance(type),
-                    runtime.GetType((TypeImpl)type),
-                    inspectionContext: CreateDkmInspectionContext(DkmEvaluationFlags.HideNonPublicMembers));
+                    runtime.GetType((TypeImpl)type));
             }
 
             var rootExpr = "new C()";
-            var evalResult = FormatResult(rootExpr, valueWithNonPublicHidden);
+            var evalResult = FormatResult(rootExpr, value, inspectionContext: CreateDkmInspectionContext(DkmEvaluationFlags.HideNonPublicMembers));
```

Comment:
It's not immediately obvious, but this wraps the Formatter instance shared by all the tests.  I could make it into a field and add a constructor to initialize it, but I'm not sure it's worth it (there appear to only be 5 redundant calls).